src/components/Button/UniversalButton/styles.ts

```diff
@@ -1,76 +1,94 @@
-import styled, { css } from "styled-components";
-import { buttonSizes, buttonVariants, buttonColors, defaultButtonSize } from "@/assets/styles/constants/buttonConstants";
-import { ButtonSize } from "@/assets/styles/constants/buttonConstants";
-import {COLORS} from "@/assets/styles/constants/colors.ts";
-import {SIZES} from "@/assets/styles/constants/sizes.ts";
-import {FONTS} from "@/assets/styles/constants/fonts.ts";
+import styled, { css, RuleSet } from 'styled-components';
+
+import {
+  buttonSizes,
+  buttonVariants,
+  buttonColors,
+  defaultButtonSize,
+  ButtonSize,
+} from '@/assets/styles/constants/buttonConstants';
+import { COLORS } from '@/assets/styles/constants/colors.ts';
+import { FONTS } from '@/assets/styles/constants/fonts.ts';
+import { SIZES } from '@/assets/styles/constants/sizes.ts';
 
 type ButtonVariantType = keyof typeof buttonVariants;
 type ButtonColorType = keyof typeof buttonColors;
 
+const paddingHorizontal = '16px';
+const borderRadius = '8px';
+const outlineWidth = '3px';
+const opacityDisabled = 0.6;
```

Comment:
this variable is multiple files, why you doing this duplication, move it to some general file